#==========================================================
#
# Name: ldid - Don't use Git to set LDID_VERSION
# Date: 2023-09-14
#
# Summary:
#   Using Git to set LDID_VERSION fails, as Git fails to
#   fetch tags from a Github archive. Even when passing
#   -DLDID_VERSION, this snippet makes the build fail.
#
#   It's best to just remove it.
#
#==========================================================
--- a/CMakeLists.txt	2023-09-14 22:53:24.000000000 -0400
+++ b/CMakeLists.txt	2023-09-14 22:52:50.000000000 -0400
@@ -21,24 +21,6 @@
     set(LDID_LIBS ${PLIST_LIBRARIES} ${CRYPTO_LIBRARIES})
 endif()
 
-find_package(Git)
-if (!Git_FOUND)
-    message(FATAL_ERROR "Git not found!")
-endif()
-
-execute_process(
-    COMMAND "${GIT_EXECUTABLE}" describe --tags --match=v*
-    WORKING_DIRECTORY "${CMAKE_SOURCE_DIR}"
-    COMMAND_ERROR_IS_FATAL ANY
-    ERROR_QUIET
-    RESULT_VARIABLE res
-    OUTPUT_VARIABLE LDID_VERSION
-    OUTPUT_STRIP_TRAILING_WHITESPACE)
-
-string(REGEX REPLACE "-([^-]*)-([^-]*)$" "+\\1.\\2" LDID_VERSION ${LDID_VERSION})
-string(REGEX REPLACE "^v" "" LDID_VERSION ${LDID_VERSION})
-string(REGEX REPLACE "%" "~" LDID_VERSION ${LDID_VERSION})
-
 add_executable(ldid ldid.cpp lookup2.c)
 
 set_property(SOURCE ldid.cpp APPEND
